### PR TITLE
Add Azure v2 blog

### PR DESCRIPTION
**Pull request by mnlumi**
## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Comment by joeduffy** [comment]
Should we start with a "reminder of why Azure Native is so great" at the top? I also always loved the visual "Full API Coverage" chart from https://www.pulumi.com/blog/announcing-nextgen-azure-provider/.

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- type: azure-native:eventgrid:Topic
++ type: azure-native:eventgrid/v20200601:Topic
+```
+
+{{% /choosable %}}
+{{< /chooser >}}
+
+## Expanded Resource Coverage
+
+We're excited to introduce [over 100 new Azure resources](link TODO) in this release, expanding the coverage of our Azure Native Provider bringing the total to over ### <!-- TODO get most recent number -->. With this extensive collection of resources at your disposal, you can now easily provision and manage an even broader range of Azure services using Pulumi. Whether you're working with virtual machines, databases, networking, or any other Azure service, the Azure Native Provider has you covered.
```
Comment (scottslowe) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:89]: Just flagging this to remind of the TODO needed here before it goes live

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+## Resolved Title Case Inconsistency
+
+In our commitment to excellence, we have [resolved title case inconsistencies](https://github.com/pulumi/pulumi-azure-native/pull/2366) across resource types and properties in the Azure Native Provider. All previous resource names have been aliased, so no additional modifications are required for existng Pulumi programs. This enhancement ensures a unified and predictable experience while switching between Azure and Pulumi resource references, making it easier for you to navigate and understand the provider's capabilities.
+
+## Simplified User Assigned Identity Inputs
+
+We have simplified the representation of user assigned identity inputs by recognizing them as string arrays. This change provides a more intuitive and clear approach when configuring user assigned identities for your Azure resources. You can now reference these as `#TODO example reference` without jumping through any hoops.
```
Comment (scottslowe) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:97]: Flagging the TODO to be fixed before going live

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+## Migrating from 1.0 to 2.0
+
+We recommend you update to the [latest version of Azure Native 1.0](https://github.com/pulumi/pulumi-azure-native/releases/tag/v1.103.0) prior to moving to 2.0. This will enable you to adress existing deprectations and see the recommended subsitutions ahead of the major version upgrade. Head over to the [migration guide](link TODO) which outlines the necessary steps to update your existing Pulumi projects and ensuring a smooth migration experience.
```
Comment (scottslowe) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:101]: One more TODO flagged as a reminder

**Review by scottslowe — COMMENTED** [review]
Good job overall; I left a few comments only to remind folks of a few TODOs needed before it goes live

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+So, how did we accomplish this? For starters, we've removed all deprecated ARM API resources <!-- TODO: Link to list of deprecated resources --> up to the 2.0 release date. The bigger impact was made by identifying and removing all forward-compatible resource versions. That is to say we've exluded any redundant versions and kept the latest version until breaking changes. Where that version is not the current latest and new default, the explicit versions are still available.
+
+![version_squeeze_diagram](./azurev2-diagrams.png)
+
+## Refreshed Default Resource Versions
+
+In the latest release, we have refreshed the default resource versions to align with the latest ARM API versions from Microsoft. With our continued committment to same-day access to Azure API surface, you always have access to the most up-to-date features and capabilities offered by Azure resources. By leveraging the latest resource versions, you can take advantage of the latest and most desired features directly from Azure while building your infrastructure with Pulumi.
```
Comment (danielrbradley) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:39]: Perhaps we need a paragraph before this explaining what "default versions" are? Something like this maybe.

> When Azure releases new features they often create a new version of a service. These versions named using dates such as "2021-01-02" or "2022-05-08-preview". There's many versions of each resource in the SDK, so we pick a default version of each resource 

I've also found this page hidden in the registry docs which we should also reference: https://www.pulumi.com/registry/packages/azure-native/version-guide/

Interestingly that page uses the termonology of "Top-Level Modules" and "Module Per Version". Perhaps we should adjust our language to match with that?

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+## Reduced SDK Size for Faster Deployments
+
+We've listened to your feedback and have made significant optimizations to the Azure Native Provider SDKs. With careful fine-tuning and restructuring, we have achieved a remarkable decrease in SDK size of up to 60% <!-- TODO: Get latest size updates on alpha.3 once released -->. This improvement translates into faster deployment times, reduced resource consumption, and an overall smoother experience when working with the provider.
+
+So, how did we accomplish this? For starters, we've removed all deprecated ARM API resources <!-- TODO: Link to list of deprecated resources --> up to the 2.0 release date. The bigger impact was made by identifying and removing all forward-compatible resource versions. That is to say we've exluded any redundant versions and kept the latest version until breaking changes. Where that version is not the current latest and new default, the explicit versions are still available.
+
+![version_squeeze_diagram](./azurev2-diagrams.png)
```
Comment (mikhailshilkov) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:35]: The numbers don't quite check out in my head... Why is Go still almost 500 MB?
And is Java really so tiny? I know we only ship default versions, but this looks too small still?

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+{{% choosable language yaml %}}
+
+```yaml
+- type: azure-native:eventgrid:Topic
++ type: azure-native:eventgrid/v20200601:Topic
+```
+
+{{% /choosable %}}
+{{< /chooser >}}
+
```
Comment (mikhailshilkov) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:86]: Should we call out the path conflict resolution that @thomas11 did? Or maybe the effect of it - e.g. that the default version of MySQL/Postgres is pointing to Flexible Servers.

**Review by mikhailshilkov — COMMENTED** [review]
Looks good overall, took the first pass on comments

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+## Migrating from 1.0 to 2.0
+
+We recommend you update to the [latest version of Azure Native 1.0](https://github.com/pulumi/pulumi-azure-native/releases/tag/v1.103.0) prior to moving to 2.0. This will enable you to adress existing deprectations and see the recommended subsitutions ahead of the major version upgrade. Head over to the [migration guide](link TODO) which outlines the necessary steps to update your existing Pulumi projects and ensuring a smooth migration experience.
+
+Current users of the Pulumi Azure Classic Provider can follow the [classic to native migration guide](https://www.pulumi.com/registry/packages/azure-native/from-classic/) to take advantage of all the latest and greatest features available in Pulumi the Azure Native Provider.
+
+While you can immediately start leveraging the improved performance, you may need to make some changes in order to continue using specific ARM API resource versions or user assigned identity inputs. Use the Pulumi Azure Native Provider documenation within your IDE to determine the explicit version that is available and fully compatible.
```
Comment (kpitzen) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:105]:
```suggestion
While you can immediately start leveraging the improved performance, you may need to make some changes in order to continue using specific ARM API resource versions or user assigned identity inputs. Use the Pulumi Azure Native Provider documentation within your IDE to determine the explicit version that is available and fully compatible.
```

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+## Reduced SDK Size for Faster Deployments
+
+We've listened to your feedback and have made significant optimizations to the Azure Native Provider SDKs. With careful fine-tuning and restructuring, we have achieved a remarkable decrease in SDK size of up to 60% <!-- TODO: Get latest size updates on alpha.3 once released -->. This improvement translates into faster deployment times, reduced resource consumption, and an overall smoother experience when working with the provider.
+
+So, how did we accomplish this? For starters, we've removed all deprecated ARM API resources <!-- TODO: Link to list of deprecated resources --> up to the 2.0 release date. The bigger impact was made by identifying and removing all forward-compatible resource versions. That is to say we've exluded any redundant versions and kept the latest version until breaking changes. Where that version is not the current latest and new default, the explicit versions are still available.
```
Comment (kpitzen) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:33]:
```suggestion
So, how did we accomplish this? For starters, we've removed all deprecated ARM API resources <!-- TODO: Link to list of deprecated resources --> up to the 2.0 release date. The bigger impact was made by identifying and removing all forward-compatible resource versions. That is to say we've excluded any redundant versions and kept the latest version until breaking changes. Where that version is not the current latest and new default, the explicit versions are still available.
```

File: themes/default/content/blog/introducing-azure-native-v2/index.md (hunk 4 above)
Comment (kpitzen) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:39]:
```suggestion
In the latest release, we have refreshed the default resource versions to align with the latest ARM API versions from Microsoft. With our continued commitment to same-day access to Azure API surface, you always have access to the most up-to-date features and capabilities offered by Azure resources. By leveraging the latest resource versions, you can take advantage of the latest and most desired features directly from Azure while building your infrastructure with Pulumi.
```

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+{{% /choosable %}}
+{{< /chooser >}}
+
+## Expanded Resource Coverage
+
+We're excited to introduce [over 100 new Azure resources](link TODO) in this release, expanding the coverage of our Azure Native Provider bringing the total to over ### <!-- TODO get most recent number -->. With this extensive collection of resources at your disposal, you can now easily provision and manage an even broader range of Azure services using Pulumi. Whether you're working with virtual machines, databases, networking, or any other Azure service, the Azure Native Provider has you covered.
+
+## Resolved Title Case Inconsistency
+
+In our commitment to excellence, we have [resolved title case inconsistencies](https://github.com/pulumi/pulumi-azure-native/pull/2366) across resource types and properties in the Azure Native Provider. All previous resource names have been aliased, so no additional modifications are required for existng Pulumi programs. This enhancement ensures a unified and predictable experience while switching between Azure and Pulumi resource references, making it easier for you to navigate and understand the provider's capabilities.
```
Comment (kpitzen) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:93]:
```suggestion
In our commitment to excellence, we have [resolved title case inconsistencies](https://github.com/pulumi/pulumi-azure-native/pull/2366) across resource types and properties in the Azure Native Provider. All previous resource names have been aliased, so no additional modifications are required for existing Pulumi programs. This enhancement ensures a unified and predictable experience while switching between Azure and Pulumi resource references, making it easier for you to navigate and understand the provider's capabilities.
```

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+
+{{% notes type="warning" %}}
+The Pulumi Azure Native Provider 2.0 is currently available in beta.
+{{% /notes %}}
+
+We are thrilled to announce the highly anticipated beta release of the Pulumi Azure Native Provider 2.0, a significant upgrade to our infrastructure as code (IaC) solution for Microsoft Azure. This release brings a host of exciting features and improvements that will enhance your experience with managing Azure resources and empower you to build robust and scalable cloud infrastructure more efficiently.
+
+<!--more-->
+
+At Pulumi, we understand the importance of keeping up with the ever-evolving cloud landscape. The Pulumi Azure Native Provider 2.0 represents our commitment to providing you with the best tools to harness the full power of Microsoft Azure while providing you with same-day access to the entire [Azure API surface](https://docs.microsoft.com/en-us/rest/api/azure/). Let's dive into some of the key enhancements in this release:
```
Comment (thomas11) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:27]: What do we mean here by "same-day"? We don't release every day so it's not true that additions to the Azure API spec are available on the same day.

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+
+## Reduced SDK Size for Faster Deployments
+
+We've listened to your feedback and have made significant optimizations to the Azure Native Provider SDKs. With careful fine-tuning and restructuring, we have achieved a remarkable decrease in SDK size of up to 60% <!-- TODO: Get latest size updates on alpha.3 once released -->. This improvement translates into faster deployment times, reduced resource consumption, and an overall smoother experience when working with the provider.
```
Comment (thomas11) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:31]: I would expect deployment times to stay pretty much the same, no? Unless you're memory-constrained. I think the speed-up at development time and build time is more significant.

File: themes/default/content/blog/introducing-azure-native-v2/index.md (hunk 8 above)
Comment (thomas11) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:33]:
```suggestion
So, how did we accomplish this? For starters, we've removed all deprecated ARM API resources <!-- TODO: Link to list of deprecated resources --> up to the 2.0 release date. The bigger impact was made by identifying and removing all forward-compatible resource versions. That is to say we've excluded any redundant versions and kept only the versions that introduce breaking changes. Where these versions are not the current latest and new default, they remain available as explicit versions.
```

Comment (thomas11) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:33]: I think we should define what "default" and "explicit" resources/versions are.

File: themes/default/content/blog/introducing-azure-native-v2/index.md (hunk 7 above)
Comment (thomas11) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:105]:
```suggestion
While you can immediately start leveraging the improved performance, you may need to make some changes in order to continue using specific ARM API resource versions or user assigned identity inputs. Use the Pulumi Azure Native Provider documenation within your IDE to determine the explicit API version that is available and fully compatible with your existing deployment.
```

File: themes/default/content/blog/introducing-azure-native-v2/index.md (hunk 10 above)
Comment (mikhailshilkov) [reply themes/default/content/blog/introducing-azure-native-v2/index.md:27]: TL;DR: same-day is okay.
We discussed this a few times and the truth is that "we can ship our provider on the same day when API specs are published". In reality, the day when specs are published is fairly meaningful (except for major launches, when we can cut a special release if needed): specs may ship after the service is available, or even before it's available, and service deployment takes days-to-weeks across regions anyway. That's too advanced message, so we stick to "same-day" as mostly a marketing message but with substance that "we can when needed".

File: themes/default/content/blog/introducing-azure-native-v2/index.md (hunk 11 above)
Comment (mikhailshilkov) [reply themes/default/content/blog/introducing-azure-native-v2/index.md:31]: Doesn't deploy time include build time?

File: themes/default/content/blog/introducing-azure-native-v2/index.md (hunk 10 above)
Comment (mnlumi) [reply themes/default/content/blog/introducing-azure-native-v2/index.md:27]: We have the ability to consume and release upstream changes same-day. We don't normally <need> to do same-day deliveries, but can. It's also possible that our release cadence falls on the same-day as upstream changes.

File: themes/default/content/blog/introducing-azure-native-v2/index.md (hunk 11 above)
Comment (mnlumi) [reply themes/default/content/blog/introducing-azure-native-v2/index.md:31]: Deployment could be faster in CI, as we've heard from feedback. I think development time is a better call out.

File: themes/default/content/blog/introducing-azure-native-v2/index.md (hunk 6 above)
Comment (mnlumi) [reply themes/default/content/blog/introducing-azure-native-v2/index.md:88]: This will be included in the migration guide. I'm okay leaving it out here.

**Comment by mnlumi** [comment]
> Should we start with a "reminder of why Azure Native is so great" at the top? I also always loved the visual "Full API Coverage" chart from https://www.pulumi.com/blog/announcing-nextgen-azure-provider/.

@joeduffy I've made some updates per your feedback.

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+    - providers
+    - azure
+
+---
+
+{{% notes type="info" %}}
+The Pulumi Azure Native Provider 2.0 is currently available in beta.
+{{% /notes %}}
+
+We are thrilled to announce the highly anticipated beta release of the Pulumi Azure Native Provider 2.0, a significant upgrade to our infrastructure as code (IaC) solution for Microsoft Azure. The native Azure provider implements the best possible support for the Azure platform in Pulumi. Every property of each module is always represented in the SDKs. This release brings a host of exciting features and improvements that will enhance your experience with managing Azure resources and empower you to build robust and scalable cloud infrastructure more efficiently.
```
Comment (lukehoban) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:23]:
```suggestion
We are thrilled to announce the beta release of the Pulumi Azure Native Provider 2.0, a significant upgrade to Pulumi's native provider for Microsoft Azure. The Azure Native provider offers the most complete support for Azure possible - with same day access to the entire surface area of the Azure features from Azure Resource Manager. Every property of each module is always represented in the SDKs. The 2.0 release brings a host of exciting features and improvements for performance and usability that will enhance your experience with managing Azure resources and empower you to build robust and scalable cloud infrastructure more efficiently.
```

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+
+<!--more-->
+
```
Comment (lukehoban) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:26]: I slightly wonder - could we benefit from having a short but "real" code snippet here just to ground this in code using Azure Native?  Not a big deal - I think it also works well without it too.

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+At Pulumi, we understand the importance of keeping up with the ever-evolving cloud landscape. The Pulumi Azure Native Provider 2.0 represents our commitment to providing you with the best tools to harness the full power of Microsoft Azure. With automatic generation from Azure API specs, we are able to provide you with same-day access to the entire [Azure API surface](https://docs.microsoft.com/en-us/rest/api/azure/) including all available properties. Let's dive into some of the key enhancements in this release:
+
+## Reduced SDK Size for Faster Development
+
+We've listened to your feedback and have made significant optimizations to the Azure Native Provider SDKs. With careful fine-tuning and restructuring, we have achieved a remarkable decrease in size over 50% in SDKs with multi-versioned modules. This improvement translates into faster development times, reduced resource consumption, and an overall smoother experience when working with the provider.
```
Comment (lukehoban) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:31]:
```suggestion
We've listened to your feedback and have made significant optimizations to the Azure Native Provider SDKs. With careful fine-tuning and restructuring, we have achieved a significant decrease in size of over 50% for SDKs with multi-versioned modules. This improvement translates into faster development times, reduced resource consumption, and an overall smoother experience when working with the provider.
```

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 15]
```diff
@@ -0,0 +1,113 @@
...
+
+## Expanded Top-Level Default Module Coverage
+
+We're excited to expand our top-level default version coverage by 226 resources, 274 functions, and 5526 input properties in this release, bringing the total to 1,637, 2,015, and 30,652 respectively. With this extensive collection of modules at your disposal, you can easily provision and manage the full range of Azure services using Pulumi. Whether you're working with virtual machines, databases, networking, or any other Azure service, the Azure Native Provider has you covered.
+
+![provider-coverage-by-version](./azurev2-provider-coverage.png)
+
+## Resolved Title Case Inconsistency
+
+In our commitment to excellence, we have [resolved title case inconsistencies](https://github.com/pulumi/pulumi-azure-native/pull/2366) across module types and properties in the Azure Native Provider. All previous module names have been aliased, so no additional modifications are required for existing Pulumi programs. This enhancement ensures a unified and predictable experience while switching between Azure and Pulumi module references, making it easier for you to navigate and understand the provider's capabilities.
```
Comment (lukehoban) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:97]:
```suggestion
For a consistent and predictable experience across the provider, we have resolved [an issue with title case inconsistencies](https://github.com/pulumi/pulumi-azure-native/pull/2366) across module types and properties. All previous module names have been aliased, so no additional modifications are required for existing Pulumi programs. This enhancement ensures a unified and predictable experience while switching between Azure and Pulumi module references, making it easier for you to navigate and understand the provider's capabilities.
```

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+
+## Simplified User Assigned Identity Inputs
+
+We have simplified the representation of user assigned identity inputs by recognizing them as string arrays. This change provides a more intuitive and clear approach when configuring user assigned identities for your Azure modules. You can now reference these as `UserAssignedIdentities: [exampleUserAssignedIdentity.id]` without jumping through any hoops.
```
Comment (lukehoban) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:101]: Do we have any larger examples of this?  Even something in the registry we could link to?  Might be nice just to provide a little more context.  I recall this was one of the most requested issues for the Azure Native provider.

Comment (mnlumi) [reply themes/default/content/blog/introducing-azure-native-v2/index.md:101]: Added. Will add a chooser later to include other languages.

File: themes/default/content/blog/introducing-azure-native-v2/index.md (hunk 13 above)
Comment (mnlumi) [reply themes/default/content/blog/introducing-azure-native-v2/index.md:26]: Added a code snippet to the user assigned identities section.

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 17]
```diff
@@ -0,0 +1,147 @@
+---
+title: "Introducing Azure Native 2.0"
+
+date: 2023-06-14T14:00:00-04:00
```
Comment (mikhailshilkov) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:4]: Needs a bump to today

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+
+<!--more-->
+
+At Pulumi, we understand the importance of keeping up with the ever-evolving cloud landscape. The Pulumi Azure Native Provider 2.0 represents our commitment to providing you with the best tools to harness the full power of Microsoft Azure. With automatic generation from Azure API specs, we are able to provide you with same-day access to the entire [Azure API surface](https://docs.microsoft.com/en-us/rest/api/azure/) including all available properties. Let's dive into some of the key enhancements in this release:
```
Comment (mikhailshilkov) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:27]: Is `:` at the end intentional?

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 19]
```diff
@@ -0,0 +1,147 @@
...
+
+So, how did we accomplish this? For starters, we've removed all deprecated Azure API modules up to the 2.0 release date. The bigger impact was made by identifying and removing all forward-compatible module versions. That is to say we've excluded any redundant versions and kept only the versions that introduce breaking changes. Where these versions are not the current latest and new default, they remain available as explicit versions.
+
+You may be wondering about the significance between explicit and default versions. Well, when Azure releases new features they often create a new version of a service. These versions are named using dates such as "2021-01-02" or "2022-05-08-preview". There's many versions of each module in the SDK, so we pick a version of each module for the top-level default. To learn more about module versions, visit the [Azure Native Version Guide](https://www.pulumi.com/registry/packages/azure-native/version-guide/).
```
Comment (mikhailshilkov) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:35]: The service exists in just a single instance.

```suggestion
You may be wondering about the significance between explicit and default versions. Well, when Azure releases new features they often create a new version of a service API. These versions are named using dates such as "2021-01-02" or "2022-05-08-preview". There's many versions of each module in the SDK, so we pick a version of each module for the top-level default. To learn more about module versions, visit the [Azure Native Version Guide](https://www.pulumi.com/registry/packages/azure-native/version-guide/).
```

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 20]
```diff
@@ -0,0 +1,147 @@
...
+
+![sdk-size-version-diagram](./azurev2-diagrams.png)
```
Comment (mikhailshilkov) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:37]: Nit, feel free to ignore: I would put the image before the previous paragraph.

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 21]
```diff
@@ -0,0 +1,147 @@
...
+
+## Refreshed Default Module Versions
+
+In the latest release, we have refreshed the default module versions to align with the latest Azure API versions from Microsoft. With our continued commitment to same-day access to Azure API surface, you always have access to the most up-to-date features and capabilities offered by Azure modules. By leveraging the latest module versions, you can take advantage of the latest and most desired features directly from Azure while building your infrastructure with Pulumi.
```
Comment (mikhailshilkov) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:41]:
```suggestion
In the latest release, we have refreshed the default module versions to align with the latest Azure API versions from Microsoft. With our continued commitment to same-day access to Azure API surface, you always have access to the most up-to-date features and capabilities offered by Azure services. By leveraging the latest module versions, you can take advantage of the latest and most desired features directly from Azure while building your infrastructure with Pulumi.
```

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 22]
```diff
@@ -0,0 +1,147 @@
...
+```
+
+{{% /choosable %}}
+{{< /chooser >}}
+
+## Expanded Top-Level Default Module Coverage
+
+We're excited to expand our top-level default version coverage by 226 resources, 274 functions, and 5526 input properties in this release, bringing the total to 1,637, 2,015, and 30,652 respectively. With this extensive collection of modules at your disposal, you can easily provision and manage the full range of Azure services using Pulumi. Whether you're working with virtual machines, databases, networking, or any other Azure service, the Azure Native Provider has you covered.
+
+![provider-coverage-by-version](./azurev2-provider-coverage.png)
```
Comment (mikhailshilkov) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:93]: Cool chart! Great work!

**Review by mikhailshilkov — APPROVED** [review]
🎉

File: themes/default/content/blog/introducing-azure-native-v2/index.md (hunk 17 above)
Comment (mnlumi) [reply themes/default/content/blog/introducing-azure-native-v2/index.md:4]: This is the planned time for today. 6/14 at 2pm ET, 11AM PT

**Review by kpitzen — APPROVED** [review]
This is all looking really great - excellent work @mnlumi and the rest of the team!

**Review by scottslowe — APPROVED** [review]

**Pull request merged**